### ISSUE TITLE
Does CVE-2019-20372 affect 1.16.1-1?

### ISSUE THREAD

**Issue by ingokofler** [nginxinc/docker-nginx-unprivileged#30]
According to https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=948579 it seems that the current image tagged with 1.16 is vulnerable to that CVE.

```
docker run -it --entrypoint=/bin/sh nginxinc/nginx-unprivileged:1.16
$ nginx -v
nginx version: nginx/1.16.1
$ dpkg -s nginx | grep Version
Version: 1.16.1-1~buster
```

Are there plans to fix this?
How is the release process in general for these nginx-unprivileged Docker images?

**Comment by alessfg** [comment]
There's no plans to release a new version of stable NGINX for the time being. I would recommend using the latest mainline NGINX release if you want to use a version that addresses that CVE.

**Comment by ingokofler** [comment]
@alessfg Thanks for the clarification.  Just one final question. According to https://www.nginx.com/blog/nginx-1-16-1-17-released/ it states that 1.16 will get security updates. 

"Stable receives fixes for high‑severity bugs, but is not updated with new features"

Is there any threshold for the CVE score or is this decided on a case-by-case basis?

**Comment by alessfg** [comment]
It is indeed decided on a case-by-case basis and it's been determined that that CVE is not a high-severity CVE.
